Extension/src/LanguageServer/client.ts

```diff
@@ -967,6 +969,8 @@ export class DefaultClient implements Client {
                 }
             }
         }
+        const compilerCountString: string = compilerCount.toString();
+        telemetry.logLanguageServerEvent('compilerCount', { compilerCountString });
```

Comment:
Can you add this to the telemetry properties for the 'compilerSelection' event instead of having a separate event for it?

Reply:
What is the syntax for that?

Reply:
The syntax is delete this line and the line above and add it to the "measures" (3rd optional argument which accepts numbers instead of strings for the values).

Something like this (I didn't compile)
```ts
telemetry.logLanguageServerEvent('compilerSelection', { action }, { compilerCount });
```